npc/re/jobs/novice/academy.txt

```diff
@@ -13,80 +13,245 @@
-			mes "Had a good dream?";
-			mes "Soon, we will get to ^4d4fffIzlude^000000.";
-			next;
-			mes "[Captain Carocc]";
-			mes "And you can talk to other people like you just talked to me.";
-			next;
-			cutin "tutorial01",3;
-			mes "!- Information -!";
-			mes "^4d4fffLeft mouse click allows basic game controls.";
-			mes "Such as conversation with NPC, movement, and attacks.^000000";
-			next;
-			cutin "fly_trock",2;
-			mes "[Captain Carocc]";
-			mes "Many youngsters visit the ^4d4fffRune Midgard Kingdom^000000 to chase their dreams.";
-			mes "Izlude is a satellite city of Prontera, the capital of the Rune Midgard Kingdom.";
-			next;
-			mes "[Captain Carocc]";
-			mes "Izlude always welcomes young adventurers like you who have just started their journey to the World.";
-			mes "You are one of them of course.";
+//== Starting Point  (iz_int) =================================
+iz_int,18,26,0	script	startpoint	HIDDEN_WARP_NPC,1,1,{
+OnTouch:
+	if (!izintspawn) {
+		specialeffect EF_ANGEL3, AREA, playerattached(); // On official it is some kind of Poring Angel, but I can't find it
+		navigateto("int_land", NAV_NONE, 1); // individual map name not required
+		izintspawn = 1;
+	}
+	end;
+}
+iz_int01,18,27,0	duplicate(startpoint)	startpoint#01	HIDDEN_WARP_NPC,1,1
+iz_int02,18,27,0	duplicate(startpoint)	startpoint#02	HIDDEN_WARP_NPC,1,1
+iz_int03,18,27,0	duplicate(startpoint)	startpoint#03	HIDDEN_WARP_NPC,1,1
+iz_int04,18,27,0	duplicate(startpoint)	startpoint#04	HIDDEN_WARP_NPC,1,1
+
+//== Wounded Swordsman (iz_int) =================================
+iz_int,56,32,3	script	Wounded Swordsman#izint	4_TOWER_01,8,8,{
+	if (!@display) {
+		classchange(4_TOWER_02, 0);
+		npctalk "Ohhhhhhh....";
+		@display = 1;
+		end;
+	}
+	if(checkquest(21001) == -1) {
+		mes "[Wounded]";
+			mes "Wow! Thank you a lot!";
+			mes "I don't know how this happend to our ship";
+			mes "but we should go to see the captain.";
 			next;
-			mes "[Captain Carocc]";
-			mes "I would like to tell you more stories, but it's time to get ready for arrival.";
-			mes "There is an academy in Izlude for newcomers like yourself.";
+			mes "[Wounded]";
+			mes "... ohh, it seems my body is injured too much.";
+			mes "Maybe you can go without me?";
+			setquest(21001);
+			close;
+	}
+	else {
+		mes "[Wounded]";
+			mes "Luckily the water does not occupy the ship.";
+			mes "I wait here, you should go and see the captain.";
 			next;
+			mes "[Wounded]";
+			mes "I'm fine! I'm fine!";
+			mes "Don't worry about me, you should really go to the captain!";
+	}
+	cutin "tutorial02", 4;
+	mes "^4d4dff!- Information -!^000000";
+	mes "NPC Quest Received.";
+	mes "^4d4dffQuestinfo Shortcut is Alt + U^000000 You can check your quest status there anytime.";
+	close2;
+	cutin "", 255;
+	end;
+
+OnInit:
+	questinfo 21001, QTYPE_QUEST, 1, Job_Novice;
+	end;
+
+OnTouch_:
+	if (@cd + 15 > gettimetick(2))
+		end;
+	@cd = gettimetick(2);
+	npctalk "Urgh... please... help me...";
```

Comment:
They used `@variable`, attached to the character. `npctalk` is probably for self.
`npctalk "Urgh... please... help me...", "", bc_self;`